packages/remix-server-runtime/server.ts

```diff
@@ -335,7 +335,7 @@ async function handleDocumentRequestRR(
       ...entryContext,
       staticHandlerContext: context,
       serverHandoffString: createServerHandoffString({
-        url: context.location.pathname + context.location.search,
+        url: context.location.pathname,
```

Comment:
Keeping the `url` name in this handoff string since it's technically exposed to users via `EntryContext.serverHandoffString` and `window.__remixContext` so just in case anyone is accessing it.